### ISSUE TITLE
Quickstart just don't work for minikube

### ISSUE THREAD

**Issue by staticdev** [elastic/cloud-on-k8s#934]
## Bug Report
**What did you do?**

Just following the [quickstart guide](https://www.elastic.co/guide/en/cloud-on-k8s/current/index.html) on minikube 1.1.0 (kubernetes 1.14.2).

I just deployed the Elasticsearch and it went green health and operational.

**What did you expect to see?**

https://quickstart-es:9200 -> see a website.

**What did you see instead? Under which circumstances?**

Server not found.

I think some minikube instructions need to be added (the same way there are instructions for the Helm chart) since many people will first try on a development machine instead of production cluster.

**Environment**

Linux with kvm2

* Version information:

      0.8.0

	insert version or git commit id here

* Kubernetes information:

	insert any information about your Kubernetes environment that could help us:
	* On premise ? Yes. Development machine Minikube.
	* Cloud: GKE / EKS / AKS ? Not yet.
	* Kubernetes distribution: Openshift / Rancher / PKS ? Minikube 1.1.0.
	
	for each of them please give us the version you are using
```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.2", GitCommit:"66049e3b21efe110454d67df4fa62b08ea79a19b", GitTreeState:"clean", BuildDate:"2019-05-16T16:23:09Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.2", GitCommit:"66049e3b21efe110454d67df4fa62b08ea79a19b", GitTreeState:"clean", BuildDate:"2019-05-16T16:14:56Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
```

* Resource definition:
```
-
```

* Logs:
```
No need for logs, minikube just don't show this kind of url on the browser. You need to use minikube service.
```

**Comment by thbkrkr** [comment]
If you open `https://quickstart-es:9200`  in your browser, it's normal that it does not work.

`https://quickstart-es:9200` will work only from inside the Kubernetes cluster. 

To test it, you can execute `curl` from a container:
```
> kubectl run --rm -ti --image=krkr/debug bash         
/ # curl -sk https://quickstart-es:9200 | jq .status
401
```

Otherwise run `kubectl port-forward service/quickstart-es 9200` to be able to use `https://localhost:9200` from your local workstation.

**Comment by staticdev** [comment]
@thbkrkr I also tried the port-forward method from the docs and localhost:9200. It doesn't work with minikube.

**Comment by thbkrkr** [comment]
> @thbkrkr I also tried the port-forward method from the docs and localhost:9200. It doesn't work with minikube.

What's the output of the `kubectl port-forward svc/quickstart-es 9200` command?

```shell
> kubectl port-forward svc/quickstart-es 9200
Forwarding from 127.0.0.1:9200 -> 9200
Forwarding from [::1]:9200 -> 9200
```

Do you keep the command active in a separate shell from the one used to execute `curl`?
In another shell:
```shell
> curl -k https://localhost:9200
{"error":{"root_cause":[{"type":"security_exception","reason":"missing authentication credentials for REST request [/]","header":{"WWW-Authenticate":["Bearer realm=\"security\"","ApiKey","Basic realm=\"security\" charset=\"UTF-8\""]}}],"type":"security_exception","reason":"missing authentication credentials for REST request [/]","header":{"WWW-Authenticate":["Bearer realm=\"security\"","ApiKey","Basic realm=\"security\" charset=\"UTF-8\""]}},"status":401}
```

It's normal to get a `401` HTTP error because basic authentication is configured by default. The quickstart indicates how to retrieve credentials to be authenticated.

> 
> Your kubectl command gave this result:
> 
> ```
> kubectl run --rm -ti --image=krkr/debug bash         
> kubectl run --generator=deployment/apps.v1 is DEPRECATED and will be removed in a future version. Use kubectl run --generator=run-pod/v1 or kubectl create instead.
> 
> If you don't see a command prompt, try pressing enter.
> / # / # curl -sk https://quickstart-es:9200 | jq .status
> /bin/sh: /: Permission denied
> / # 401
> /bin/sh: 401: not found
> / # 
> / # 
> ```

`/ #` is not part of the command. Once in the container, simply execute `curl -sk https://quickstart-es:9200 | jq .status`.

**Comment by staticdev** [comment]
Nice.. I got the 401 from curl and did an authenticated curl. But I really think the quick guide misled me into testing with the browser.

I would put this after this curl test:

>  Get access to Elasticsearch.
> 
> From the Kubernetes cluster, use this URL:
> 
> https://quickstart-es:9200

**Comment by pebrc** [comment]
Closing this as it seems have been addressed with #978
